tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Extensions/CreateOrderItemModelExtensionsTests.cs

```diff
@@ -33,6 +33,18 @@ static void Test()
             Assert.Throws<ArgumentNullException>(Test);
         }
 
+        [Test]
+        public void ToRequest_NullQuantity_ThrowsArgumentException()
+        {
+            static void Test()
+            {
+                CreateOrderItemModelBuilder.Create().WithQuantity(null).Build()
+                    .ToRequest(OrderBuilder.Create().Build(), CatalogueItemType.Solution);
+            }
+
+            Assert.Throws<ArgumentException>(Test);
```

Comment:
Would you like to revisit this test since the discussions in the coding standards meeting?